http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/NettyNioAsyncHttpClient.java

```diff
@@ -385,7 +386,7 @@ public String clientName() {
         private Integer maxHttp2Streams;
         private SslProvider sslProvider;
         private ProxyConfiguration proxyConfiguration;
-        private TlsKeyManagersProvider tlsKeyManagersProvider = SystemPropertyTlsKeyManagersProvider.create();
+        private TlsKeyManagersProvider tlsKeyManagersProvider = NONE_PROVIDER;
```

Comment:
Why not just use 'null'?

Reply:
The idea is to let the customer explicitly set to `null` so no resolution happens for this client, even if the properties are set.